src/common/utility.c

```diff
@@ -995,6 +995,29 @@ gchar *dt_str_replace(const char *string, const char *search, const char *replac
   return res;
 }
 
+gboolean dt_str_commasubstring(const char *list, const char *search)
+{
+  if(search == NULL)
+    return FALSE;
+
+  gchar *nlist = g_strdup(list);
+  char delimiter[] = ",";
+
+  char *ptr =strtok(nlist, delimiter);
+  while(ptr != NULL)
+  {
+    if(g_strcmp0(search, ptr) == 0)
+    {
+      g_free(nlist);
+      return TRUE;
+    }
+	  ptr = strtok(NULL, delimiter);
```

Comment:
Indentation seems wrong.

Reply:
To simplify you could record the module list with an added coma at the end. And so the check needs just to strtok() search catenated with a coma. It will work for all module names included the last one. But I won't force you doing this of course... Just in case you agree that it may be simpler.

Reply:
Last suggestion would need a str duplication as we can't just take what we get by `argv[++k]` and a `g_free` at closing No?

As you know, i am definitely not the C expert :-)

Reply:
Right, it would need a string duplication. And BTW, you need to add a coma before and after the list to check for `,module,` to avoid any miss match. Maybe not worth it :)

Reply:
:-)